docker-compose.yml

```
@@ -70,3 +70,13 @@ services:
         RAILS_VERSION: 4.2
     volumes:
       - .:/source:delegated
+
+  ruby-2.3-rails-4.1:
+    build:
+      context: .
+      dockerfile: dockerfiles/ruby23.dockerfile
```

Could you use `2.3` instead of `23` because we use `X.Y` style in other places?

```suggestion
      dockerfile: dockerfiles/ruby2.3.dockerfile
```

---

Done.